meshpy/mesh.py

```diff
@@ -632,6 +642,100 @@ def couple_nodes(
             for node_list in partner_nodes:
                 self.add(coupling_factory(node_list, coupling_type, coupling_dof_type))
 
+    def interwove_nodes_of_z_cylinder(self, *, nodes=None, fibers, beam_radius=None):
```

Comment:
This is a highly problem specific function and does not belong as a member of `Mesh`.